Android.mk

```diff
@@ -85,6 +85,8 @@ LOCAL_LDFLAGS := \
 $(LOCAL_PATH)/src/version.c: $(LOCAL_PATH)/src/version.c.in
 	sed 's/SOURCE_VERSION/"$(LOCAL_PRIVATE_SOURCE_VERSION)"/' < $< > $@
 
+NCP_SPINEL_SRC_FILES := $(wildcard $(LOCAL_PATH)/src/ncp-spinel/*.cpp) $(wildcard $(LOCAL_PATH)/src/ncp-spinel/*.c)
```

Comment:
We don't need `*.h` files?

Reply:
Android.mk is different from automake. We don't need list `*.h` files. I also removed other header files.